Intersect.Client/Interface/Game/Chat/Chatbox.cs

```diff
@@ -372,27 +372,24 @@ void ChatBoxSendBtn_Clicked(Base sender, ClickedEventArgs arguments)
 
         void TrySendMessage()
         {
-            if (mLastChatTime > Globals.System.GetTimeMs())
+            var msg = mChatboxInput.Text.Trim();
+            if (string.IsNullOrWhiteSpace(msg) || string.Equals(msg, GetDefaultInputText(), StringComparison.Ordinal))
             {
-                ChatboxMsg.AddMessage(new ChatboxMsg(Strings.Chatbox.toofast, Color.Red, ChatMessageType.Error));
-                mLastChatTime = Globals.System.GetTimeMs() + Options.MinChatInterval;
+                mChatboxInput.Text = GetDefaultInputText();
 
                 return;
             }
 
-            mLastChatTime = Globals.System.GetTimeMs() + Options.MinChatInterval;
-
-            if (mChatboxInput.Text.Trim().Length <= 0 || mChatboxInput.Text == GetDefaultInputText())
+            if (mLastChatTime > Timing.Global.Milliseconds)
```

Comment:
UTC

Reply:
Timing.Global.MillisecondsUtc  (?)
what about the one in L:386 ?